src/gatling/testdata/initial-data-load.sql

```diff
@@ -0,0 +1,11 @@
+insert into services (service_name, service_description) values ('cmc', null)
+on conflict on constraint services_pkey do update set service_description = null;
```

Comment:
any reason for updating service_description = null on conflict; ideally on conflict it should do nothing

Reply:
yes right

Reply:
done